chart/compass/charts/external-services-mock/templates/auditlog-secret.yaml

```diff
@@ -8,3 +8,15 @@ data:
   auditlog-client-id: {{ "client_id" | b64enc | quote }}
   auditlog-client-secret: {{ "client_secret" | b64enc | quote }}
   auditlog-oauth-url: {{ printf "http://compass-external-services-mock.%s.svc.cluster.local:%s/audit-log/v2/oauth/token" .Release.Namespace (.Values.service.port | toString) | b64enc | quote }}
+---
```

Comment:
Rename this file to something generic so it's unified with `chart/compass/charts/external-services-mock/templates/basic-credentials-secret.yaml`